libs/dh/shared/navigation/src/navigation.service.ts

```diff
@@ -73,7 +73,7 @@ export class DhNavigationService {
     }
 
     if (path === 'details') {
-      this.router.navigate([path, id], {
+      this.router.navigate([path, id, ...args], {
```

Comment:
question: the `args` doesn't seem to be used. Is it something coming later?

Reply:
We are now able to add as many values to the query string as we want. I had a use case for it, but refactored it to do it in another way, but decided to keep the functionality.